docs/user_guides/fs/compute_engines.md

```diff
@@ -46,3 +48,30 @@ If you are using Spark or Python within Hopsworks, there is no further configura
 ### Outside Hopsworks
 
 Connecting to the Feature Store from an external Spark cluster, such as Cloudera or Databricks, requires configuring it with the Hopsworks client jars, configuration and certificates. The [Spark integration guide](../integrations/spark.md) explains step by step how to connect to the Feature Store from an external Spark cluster.
+
+## Flink
+
+### Inside Hopsworks
+
+If you are using Flink within Hopsworks, there is no further configuration required. For more details head over to the [Getting Started Guide](https://github.com/logicalclocks/hopsworks-tutorials/tree/master/integrations/java/flink).
+
+### Outside Hopsworks
+
+Connecting to the Feature Store from an external Flink cluster, such as GCP DataProc or AWS EMR, requires Hopsworks certificates. The [Flink integration guide](../integrations/flink.md) explains step by step how to connect to the Feature Store from an external Flink cluster.
+
+## Beam
+
+### Inside Hopsworks
+
+Currently, Beam is only supported as an external client. 
+
+### Outside Hopsworks
+
+Connecting to the Feature Store from Beam DataFlowRunner, requires configuring Hopsworks certificates. The [Beam integration guide](../integrations/beam.md) explains step by step how to connect to the Feature Store from Beam Dataflow Runner.
+
+!!! warning
+Apache Beam integration with Hopsworks feature store was only tested used Dataflow Runner.
```

Comment:
You need to indent this line to so that it is displayed correctly. Check other !!!warning for the proper indentation.